### PR TITLE
Move #hide_pins_link up near pan/zoom controls

### PR DESCRIPTION
**Pull request by zarino**
An attempt at helping people click the map in very busy areas – #525.

The "Hide pins" button is more prominent now.

hide-pins

### CODE REVIEW

File: templates/web/base/maps/openlayers.html [hunk 1]
```diff
@@ -31,6 +31,11 @@
     <div id="map">
       [% INCLUDE 'maps/noscript_map.html' js = 1 %]
     </div>
+  [% IF c.req.params.no_pins %]
```
Comment (dracos) [inline templates/web/base/maps/openlayers.html]: I guess it's possible you might want to turn off pins on a `/reports` or `/report` page, but I think we probably only want this on an `/around` page :)

File: web/cobrands/sass/_base.scss [hunk 2]
```diff
@@ -1451,6 +1451,10 @@ html.js #map .noscript {
   top: 44px;
 }
 
+#hide_pins_link {
```
Comment (dracos) [inline web/cobrands/sass/_base.scss]: I think being able to hide pins on the mobile map (even as and when pagination is added) is quite important!

File: web/cobrands/sass/_layout.scss [hunk 3]
```diff
@@ -614,6 +615,34 @@ body.authpage {
   bottom: 2em;
 }
 
+.big-hide-pins-link {
+  display: block;
+  position: absolute;
+  top: 248px; // top of #fms_pan_zoom_zoomout + height of #fms_pan_zoom_zoomout + 22px gap
```
Comment (dracos) [inline web/cobrands/sass/_layout.scss]: 174+44+22 = 240, not 248?

File: web/cobrands/sass/_layout.scss [hunk 4]
```diff
@@ -614,6 +615,34 @@ body.authpage {
...
+  right: 38px; // right of #fms_pan_zoom (0.5em) + right of #fms_pan_zoom_panup (30px)
```
Comment (dracos) [inline web/cobrands/sass/_layout.scss]: I guess you could put this inside fms_pan_zoom to combat having to do this, but might not be worth it.

File: web/js/map-OpenLayers.js [hunk 5]
```diff
@@ -614,16 +614,19 @@ $.extend(fixmystreet.utils, {
             zoomToBounds( fixmystreet.markers.getDataExtent() );
         }
 
-        $('#hide_pins_link').click(function(e) {
+        $('#hide_pins_link, .big-hide-pins-link').click(function(e) {
```
Comment (dracos) [inline web/js/map-OpenLayers.js]: Is it worth having a `hide-pins-link` class on both links and then using that here/below? &lt;shrug>

File: web/cobrands/sass/_layout.scss (hunk 3 above)
Comment (zarino) [reply web/cobrands/sass/_layout.scss]: Haha! The extra `8px` is the `top` of `#fms_pan_zoom`, as you probably guessed. I left it out becuase the line was already getting pretty long, and I assumed nobody would care. Maybe I should add it in ;-)

File: web/cobrands/sass/_layout.scss (hunk 4 above)
Comment (zarino) [reply web/cobrands/sass/_layout.scss]: Don’t think it’s worth it. Also, it has nothing to do with panning or zooming :-(

**Comment by abimysoc** [comment]
There's a FAQ entry that's now potentially confusing as a result of this:

>There are too many pins on the map; I can't make a report

> Use the buttons at the bottom right of the screen – 'hide pins', or 'hide old pins', to clear the map. 

Luckily we haven't actually removed the buttons at the bottom right, just added an extra one; but let this act as a reminder to check and if necessary adjust the FAQs when making a user-facing change...

**Pull request merged**